src/http.py

```diff
@@ -1,34 +1,58 @@
 ###Finding records in Sierra that do not have working links###
 
+#10/11: formatted imports to match discussion
 import csv
+#10/11: had to pip install requests in this venv. Resolved
 import requests
 from pymarc import Field,Subfield,MARCReader
 
 class Rowish:
     def __init__(self,control,url,bib_code,suppressed,live):
```

Comment:
I don't think trailing commas are needed in your class properties. I haven't seen them being used. See example here that does not include any trailing commas:
https://docs.python.org/3/tutorial/classes.html#class-objects

Reply:
Let's remove them and commit to this PR